src/app.py

```diff
@@ -1,5 +1,7 @@
 from math import sin
 
+import plotly.graph_objects as go
```

Comment:
Please, consider chainging the go into something more meaningful. (Even something like graph_object)

Reply:
sure, but conventionally it is imported this way. I will change it for the project purposes, though.